grammar.y

```diff
@@ -1085,15 +1140,15 @@ constraint_name:
 
 primary_key_order:
```

Comment:
Gotta reflect the [changes](https://github.com/tablelandnetwork/go-sqlparser/pull/29/files#r1024527904) in the grammar